Update nf-winbase-createprivatenamespace.md

turingcompl33t opened the pull request:
Fix rendering issue.

turingcompl33t committed:
Update nf-winbase-createprivatenamespace.md

Fix rendering issue.

PRMerger14 commented:
@turingcompl33t : Thanks for your contribution! The author(s) have been notified to review your proposed change.

turingcompl33t committed:
Restore italics for "objectname"

Did not realize the intent was to italicize both "prefix" and "objectname" in initial commit.

PRMerger15 commented:
@turingcompl33t : Thanks for your contribution! The author(s) have been notified to review your proposed change.

hkhmsft committed:
Merge branch 'docs' into patch-2

hkhmsft commented:
Thanks for your contribution, I've merged your changes and they should be live soon.

PRMerger10 commented:
@lastnameholiu : Thanks for your contribution! The author(s) have been notified to review your proposed change.
